.github/workflows/frameworks.yml

```diff
@@ -1,24 +1,20 @@
 name: Frameworks
-
```

Comment:
Did you use an auto formatter script to do this?

Reply:
yes